Makefile

```diff
@@ -83,14 +92,20 @@ rgbfix: ${rgbfix_obj}
 rgbgfx: ${rgbgfx_obj}
 	$Q${CC} ${REALCFLAGS} -o $@ ${rgbgfx_obj} `${PKG_CONFIG} --libs libpng`
 
-.y.c:
-	$Q${YACC} -d ${YFLAGS} -o $@ $<
+%.y.c : %.y
+	$Q${YACC} -dt ${YFLAGS} -o $@ $<
+
+%.l.c : %.l
+	$Q${FLEX} ${LFLAGS} -o $@ $<
```

Comment:
Nothing you've done here is wrong, but `lexer.l` currently fails to build on macOS. It seems like the `noreturn` macro is causing trouble, of all things:

```
In file included from src/link/lexer.l.c:533:
/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX10.12.sdk/usr/include/unistd.h:429:18: error: 
      use of undeclared identifier 'noreturn'
void     _exit(int) __dead2;
                    ^
/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX10.12.sdk/usr/include/sys/cdefs.h:155:33: note: 
      expanded from macro '__dead2'
#define __dead2         __attribute__((noreturn))
                                       ^
include/extern/stdnoreturn.h:10:42: note: expanded from macro 'noreturn'
        #define noreturn __attribute__ ((noreturn))
                                         ^
1 error generated.
```

While this is hardly our fault (and could've been avoided quite easily!), it's likely to affect a lot of other users of rgbds.

It seems like the best solution to this (or at least a better solution than hacking in `#undef noreturn`) is to invoke flex with the `--nounistd` option. I'm not aware of what repercussions this option has, but it removes the inclusion at fault, avoiding the compilation error. Thoughts?